.zenodo.json

```diff
@@ -22,44 +27,48 @@
       "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
       "orcid": "0000-0002-8218-3116"
     },
+    {
+      "name": "Ferragina, Luca",
+      "affiliation": "CERN"
+    },
     {
       "name": "Gruber, Bernhard Manfred",
       "affiliation": "CASUS, Helmholtz-Zentrum Dresden-Rossendorf, CERN",
       "orcid": "0000-0001-7848-1690"
     },
     {
-      "name": "Huebl, Axel",
-      "affiliation": "Lawrence Berkeley National Laboratory",
-      "orcid": "0000-0003-1943-7141"
+      "name": "Kaever, Christian",
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf"
     },
     {
       "name": "Kelling, Jeffrey",
       "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
       "orcid": "0000-0003-1761-2591"
     },
     {
-      "name": "Pantaleo, Felice",
+      "name": "Martin-Haugh, Stewart",
+      "affiliation": "Science and Technology Facilities Council",
+      "orcid": "0000-0001-9457-1928"
```

Comment:
@StewMH Are your affiliation and ORCID correct here? I inferred the former from the e-mail address used in your commit(s).

Reply:
Updated to your preference.